### PR TITLE
Upgrade to qiskit-sphinx-theme 1.13

### PR DESCRIPTION
**Pull request by Eric-Arellano**
The main change is the new `qiskit` theme based on Furo, but we don't yet activate that. 

Otherwise, the only relevant change is simplifying the API for custom directives. And fixing 404 errors about missing Thebe files.

--

This also fixes the images not working for two `qiskit-card`s, which I accidentally broke in https://github.com/Qiskit/qiskit-metapackage/pull/1749. For `qiskit-card`, the images apparently must live in the `_static` folder for Sphinx to know to include the images in the final build. This is unlike when we normally use images.

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -75,34 +75,18 @@ going to run and install the packages.  There are three main ways to do this:
...
-          <div class="row">
-          <div id="tutorial-cards">
-          <div class="list">
-
-       .. qiskit-card-item::
+       .. qiskit-card::
           :header: IBM Quantum Lab
           :card_description: Build quantum applications and experiments with Qiskit in a cloud programming environment.
-          :image: images/ibm_qlab.png
+          :image: _static/images/ibm_qlab.png
```
Comment (javabster) [inline docs/getting_started.rst:81]: I think something has gone wrong here? in the html artefacts the images don't seem to be loading:

Screenshot 2023-07-12 at 9 30 18 AM

Comment (Eric-Arellano) [reply docs/getting_started.rst:81]: Ah, the files needed to be in `_static/images/` and I had them in `_static`. Good find. Fixed!

Comment (Eric-Arellano) [reply docs/getting_started.rst:81]: Screenshot 2023-07-12 at 9 53 43 AM

**Review by javabster — APPROVED** [review]
LGTM 🚀

**Pull request merged**